hack/make/.integration-daemon-start

```diff
@@ -149,3 +149,5 @@ while ! ${TEST_CLIENT_BINARY} version &> /dev/null; do
 	sleep 2
 done
 printf "\n"
+
+bundle .ensure-emptyfs
```

Comment:
Does this not need to still exclude Windows?

Reply:
If so, it could just be a simplified version of that detection script like `[ "$(docker version --format '{{ .Server.Os }}')" != 'windows' ]` (especially now that we can probably pretty reasonably assume `docker version -f` support :joy:)